test/e2e/cht-conf-utils.js

```diff
@@ -0,0 +1,57 @@
+const path = require('path');
+const { exec } = require('child_process');
+const fs = require('fs');
+const fse = require('fs-extra');
+
+const log = require('../../src/lib/log');
+const { getProjectUrl } = require('./cht-docker-utils');
+
+const getProjectDirectory = (projectName) => path.resolve(__dirname, `../../build/${projectName}`);
+
+const runChtConf = (projectName, command) => new Promise((resolve, reject) => {
+  getProjectUrl(projectName).then(url => {
+    const projectDirectory = getProjectDirectory(projectName);
+    const cliPath = path.join(__dirname, '../../src/bin/index.js');
+    exec(`node ${cliPath} --url=${url} ${command}`, { cwd: projectDirectory }, (error, stdout, stderr) => {
+      if (!error) {
+        return resolve(stdout);
+      }
+
+      log.error(stderr);
+      reject(new Error(stdout.toString('utf8')));
+    });
+  });
+});
+
+const cleanupProject = (projectName) => {
+  const projectDirectory = getProjectDirectory(projectName);
+  fse.removeSync(projectDirectory);
+};
+
+const initProject = async (projectName) => {
+  const projectDirectory = getProjectDirectory(projectName);
+  if (fs.existsSync(projectDirectory)) {
+    fse.removeSync(projectDirectory);
+  }
+
+  fse.mkdirpSync(projectDirectory);
+  fs.writeFileSync(
```

Comment:
What do we actually need this package.json file for?  The current test seems to run fine without it (but maybe you are anticipating something in the future).  

For the current script code, it seems like we don't need the cht-conf dependency here since `runChtConf` is directly calling the `index.js` file.

Reply:
I believe the only scenario where a `package.json` file would be required is if a test covers [`src/lib/check-cht-conf-dependency-version.js`](https://github.com/medic/cht-conf/blob/main/src/lib/check-cht-conf-dependency-version.js). Since it's closer to what an actual config directory looks like, I would rather keep it